core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -273,10 +273,10 @@ class HttpGoogleServicesDAO(
 
     retryWithRecoverWhen500orGoogleError(() => {
       val bucket = new Bucket().setName(bucketName)
-      val deleteAfterOneYear = new Lifecycle.Rule()
+      val storageLogExpiration = new Lifecycle.Rule()
```

Comment:
I agreed, until "one year" became "180 days" at which point the variable name became misleading. Since the time period is in config, it can't be encoded into the variable name.